### ISSUE TITLE
No violation before or after changes for static import list

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#5065]
Identified at https://github.com/checkstyle/checkstyle/pull/5053#issue-254777285 ,

specifically at the code https://github.com/checkstyle/checkstyle/commit/28e618b1f1660b976803e5fdad51562152bb8e3d#diff-f784beebc72b92982be3042b568387cfR22

With empty line:
````
$ cat TestClass.java
import static com.puppycrawl.tools.checkstyle.internal.XpathUtil.getXpathItems;

import static org.junit.Assert.assertEquals;

public class TestClass {
    void method() {
    }
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
    <module name="ImportOrder">
      <property name="groups" value="/^java\./,javax,org"/>
      <property name="ordered" value="true"/>
      <property name="separated" value="true"/>
      <property name="option" value="top"/>
      <property name="sortStaticImportsAlphabetically" value="true"/>
</module>
    </module>
</module>

$ java -jar checkstyle-8.2-all.jar -c TestConfig.xml TestClass.java
Starting audit...
Audit done.
````
Without empty line:
````
$ cat TestClass.java
import static com.puppycrawl.tools.checkstyle.internal.XpathUtil.getXpathItems;
import static org.junit.Assert.assertEquals;

public class TestClass {
    void method() {
    }
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
    <module name="ImportOrder">
      <property name="groups" value="/^java\./,javax,org"/>
      <property name="ordered" value="true"/>
      <property name="separated" value="true"/>
      <property name="option" value="top"/>
      <property name="sortStaticImportsAlphabetically" value="true"/>
</module>
    </module>
</module>

$ java -jar checkstyle-8.2-all.jar -c TestConfig.xml TestClass.java
Starting audit...
Audit done.
````

There was no violation with an empty line after `import static com.puppycrawl.tools.checkstyle.internal.XpathUtil.getXpathItems;` or when the empty line was removed in the PR.
I expected ImportOrder to enforce a specific behavior in this regards that all users should follow. I expected a violation in one of the examples, not both.

**Comment by romani** [comment]
fix is merged.

**Comment by romani** [comment]
fix for this issue is hard to simply revert, as extra fix is based on it #5176.
Conflict of interests of users demands become bigger, we should go forward with fixes (no revert).
